node/cli/src/chain_spec.rs

```diff
@@ -116,7 +116,7 @@ fn staging_testnet_config_genesis() -> GenesisConfig {
 				.collect::<Vec<_>>(),
 		}),
 		session: Some(SessionConfig {
-			keys: initial_authorities.iter().map(|x| (x.1.clone(), session_keys(x.2.clone()))).collect::<Vec<_>>(),
+			keys: initial_authorities.iter().map(|x| (x.0.clone(), session_keys(x.2.clone()))).collect::<Vec<_>>(),
```

Comment:
Do we maybe want to introduce a unit wrapper `StashAccount` or something similar that tries to prevent this stuff at compilation? 
Or at least put a comment at the `keys` config in `decl_storage!` of the session module.